contracts/multicall/GuardedMulticaller2.sol

```diff
@@ -0,0 +1,240 @@
+    // solhint-disable-next-line no-unused-vars
+    constructor(address _owner, string memory _name, string memory _version) EIP712(_name, _version) {
+        _grantRole(DEFAULT_ADMIN_ROLE, _owner);
+    }
+
+    /**
+     *
+     * @notice Execute a list of calls. Returned data from calls are ignored.
+     *  The signature must be generated by an address with MULTICALL_SIGNER_ROLE
+     *  The signature must be valid
+     *  The signature must not be expired
+     *  The reference must be unique
+     *  The reference must not be executed before
+     *  The list of calls must not be empty
+     *  The list of calls is executed in order
+     *
+     * @param _multicallSigner Address of an approved signer
+     * @param _reference Reference
+     * @param _calls List of calls
+     * @param _deadline Expiration timestamp
+     * @param _signature Signature of the multicall signer
+     */
+    // slither-disable-start low-level-calls,cyclomatic-complexity
+    // solhint-disable-next-line code-complexity
+    function execute(
+        address _multicallSigner,
+        bytes32 _reference,
+        Call[] calldata _calls,
+        uint256 _deadline,
+        bytes calldata _signature
+    ) external nonReentrant {
+        // solhint-disable-next-line not-rely-on-time
+        if (_deadline < block.timestamp) {
+            revert Expired(_deadline);
+        }
+        if (_reference == 0) {
+            revert InvalidReference(_reference);
+        }
+        if (replayProtection[_reference]) {
+            revert ReusedReference(_reference);
+        }
+        if (_calls.length == 0) {
+            revert EmptyCallArray();
+        }
+        for (uint256 i = 0; i < _calls.length; i++) {
+            if (bytes(_calls[i].functionSignature).length == 0) {
+                revert InvalidFunctionSignature(_calls[i]);
+            }
+            if (_calls[i].target.code.length == 0) {
+                revert NonContractAddress(_calls[i]);
+            }
+        }
+        if (!hasRole(MULTICALL_SIGNER_ROLE, _multicallSigner)) {
+            revert UnauthorizedSigner(_multicallSigner);
+        }
+
+        // Signature validation
+        if (
+            !SignatureChecker.isValidSignatureNow(
+                _multicallSigner,
+                _hashTypedData(_reference, _calls, _deadline),
+                _signature
+            )
+        ) {
+            revert UnauthorizedSignature(_signature);
+        }
+
+        replayProtection[_reference] = true;
+
+        // Multicall
+        for (uint256 i = 0; i < _calls.length; i++) {
+            bytes4 functionSelector = bytes4(keccak256(bytes(_calls[i].functionSignature)));
+            bytes memory callData = abi.encodePacked(functionSelector, _calls[i].data);
+            // solhint-disable avoid-low-level-calls
+            // slither-disable-next-line calls-loop
+            (bool success, bytes memory returnData) = _calls[i].target.call(callData);
+            if (!success) {
```

Comment:
I think the coverage report is indicating that this branch isn't being tested

Reply:
Similar to the comments above, there are two tests that should cover that line:
https://github.com/immutable/contracts/pull/241/files#diff-7891bd472e64eb648ea5f38d97917eacbfc82723eeb5beffe2c65ee58c4c6b30R249
https://github.com/immutable/contracts/pull/241/files#diff-7891bd472e64eb648ea5f38d97917eacbfc82723eeb5beffe2c65ee58c4c6b30R203

Thinking I'm missing something here